components/encumbered/lms/Makefile

```diff
@@ -0,0 +1,119 @@
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2020 Rouven Weiler
+#
+
+BUILD_BITS=			32
+BUILD_STYLE=			archive
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		lms
+COMPONENT_FMRI =		service/multimedia/
+COMPONENT_VERSION =		7.9.2
+COMPONENT_PROJECT_URL =		https://github.org/Logitech/slimserver.git
+COMPONENT_SUMMARY =		Logitech Media Server
+COMPONENT_LICENSE =		GPLv2
+COMPONENT_LICENSE_FILE =	$(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION =	System/File System
+
+COMPONENT_REPO_NAME =		slimserver
+COMPONENT_GIT_COMMIT_HASH =	9b9f2d148bae0021913bbe9642cce8dc83a3ad00
+COMPONENT_SRC =			$(COMPONENT_REPO_NAME)-$(COMPONENT_GIT_COMMIT_HASH)
+COMPONENT_ARCHIVE =		$(COMPONENT_REPO_NAME)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH =	sha256:0e787e633ce002f5fe59da4ba71cfbfe3b43ece5a40490b58c88c2b44061fbc5
+COMPONENT_ARCHIVE_URL =		https://github.com/Logitech/$(COMPONENT_REPO_NAME)/archive/$(COMPONENT_GIT_COMMIT_HASH).tar.gz
+
+COMPONENT_REPO_NAME_1 =		slimserver-vendor
+COMPONENT_GIT_COMMIT_HASH_1 =	6265ea5b0ab9952af187602e848bbe318f4e3cc6
+COMPONENT_SRC_1 =		$(COMPONENT_REPO_NAME_1)-$(COMPONENT_GIT_COMMIT_HASH_1)
+COMPONENT_ARCHIVE_1 =		$(COMPONENT_REPO_NAME_1)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH_1 =	sha256:055d9e31c79e683da53f09e9fb02833b29ed12d4168ddc59d46447d277992da8
+COMPONENT_ARCHIVE_URL_1 =	https://github.com/Logitech/$(COMPONENT_REPO_NAME_1)/archive/$(COMPONENT_GIT_COMMIT_HASH_1).tar.gz
+
+PERL_VERSION =			5.22
+
+BUILD_TARGET =			$(BUILD_DIR)/$(COMPONENT_NAME)/.built $(BUILD_DIR)/$(COMPONENT_NAME)-vendor/.built
+
+INSTALL_TARGET =		$(BUILD_DIR)/$(COMPONENT_NAME)/.installed
+
+include $(WS_MAKE_RULES)/common.mk
+
+BUILDME_ENV +=                  MAKE=$(GMAKE)
+BUILDME_ENV +=			PATH=$(dir $(CC)):$(SPRO_VROOT)/bin:$(PATH)
+BUILDME_ENV +=                  LANG=""
+BUILDME_ENV +=			CC="$(CC)"
+BUILDME_ENV +=			LANG=""
+BUILDME_ENV +=			CFLAGS="$(PERL_OPTIMIZE)"
+
+LMS_PERL_BUILD_ARCH =		$(shell $(PERL) -MConfig -le 'print $Config{archname}')
+LMS_INSTALL_DIR =		/opt/lms-$(COMPONENT_VERSION)
+LMS_USER =			lmsd
+LMS_GROUP =			lmsd
+
+COMPONENT_POST_UNPACK_ACTION =	($(FIND) $(COMPONENT_DIR)/$(COMPONENT_REPO_NAME)-$(COMPONENT_GIT_COMMIT_HASH)/CPAN/arch \
+                                    -mindepth 1 -maxdepth 1 ! \( -name "5." -o -name "$(PERL_VERSION)" \) -type d -exec rm -rf {} + ; \
+				$(FIND) $(COMPONENT_DIR)/$(COMPONENT_REPO_NAME)-$(COMPONENT_GIT_COMMIT_HASH)/CPAN/arch \
+                                    -mindepth 1 -name "*-*" -type d -exec rm -rf {} + ; \
+                                $(FIND) $(COMPONENT_DIR)/$(COMPONENT_REPO_NAME)-$(COMPONENT_GIT_COMMIT_HASH)/Bin \
+                                    -mindepth 1 -maxdepth 1 ! \( -name "$(PERL_BUILD_ARCH)" \) -type d -exec rm -rf {} +) 
+
+COMPONENT_POST_INSTALL_ACTION +=	$(SBINDIR)/groupadd lmsd ; \
```

Comment:
build system shouldn't add users or groups, they are delivered as manifest actions.

Reply:
So as I understand correctly: I do not need to add lmsd explicitly. As long as the manifest action knows this new user and group it will automatically added by the routine(s)?

Reply:
Hope I got the correct lines for making the manifest magic do that.
I chose uid=99 and gid=99. Seems to be free according to 'https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/doc/reserved_uids_and_gids.md'

Reply:
build system can make everything it wants on build host, but only actions from manifest are applied when package is installed.